keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -614,6 +635,46 @@ Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they relate.
 -->
 
+#### Failing delete after a condition is added
+
+Here we consider a scenario when a component fails (for example its container
+dies) between appending a pod condition and deleting the pod.
+
+In particular, scheduler can possibly decide to preempt
+a different pod the next time (or none). This would leave a pod with a
+condition that it was preempted, when it actually wasn't. This in turn
+could lead to inproper handling of the pod by the job controller.
+
+One possible solution to this issue is to introduce a controller in
+kube-controller-manager that either completes a preemption (does a delete)
+or removes the condition (or set its status to false) if the condition
+timestamp is old enough and there is no deletion timestamp.
+
+We are going to re-evaluate the need for solving this issue and the solution
+for Beta.
```

Comment:
We are doing this for alpha already, removing the condition

Reply:
Updated Graduation Criteria and the Note